src/libs/actions/User.js

```diff
@@ -787,7 +787,7 @@ function setContactMethodAsDefault(newDefaultContactMethod) {
             onyxMethod: Onyx.METHOD.MERGE,
             key: ONYXKEYS.PERSONAL_DETAILS_LIST,
             value: {
-                [newDefaultContactMethod]: {
+                [currentUserAccountID]: {
                     ...myPersonalDetails,
```

Comment:
Super NAB - i guess we don't have to spread this here anymore, right? Since we're not changing accountIDs, we would only be updating login & displayName?

Reply:
Oh yah, your're right, on it!